edgelet/edgelet-docker/src/notary.rs

```diff
@@ -0,0 +1,90 @@
+use std::path::{Path,PathBuf};
+use log::{info, debug};
+use serde_json::json;
+use std::fs;
+use std::fs::File;
+use crate::{Error,ErrorKind};
+use failure::ResultExt;
+
+pub fn notary_init(homedir : &Path, hostname : String, path : PathBuf) -> Result<(), Error> {
```

Comment:
Do we always initialize notary no matter customers specify the registry or not?

Reply:
No only if it is configured in the config.yaml. The configuration does mandate mentioning the registry hostname and the associated root CA